lib/go/thrift/socket_windows_conn.go

```diff
@@ -1,4 +1,4 @@
-// +build windows
+// +build windows wasm
```

Comment:
with this change it's better to rename the file as well. maybe `socket_non_unix_conn.go`?

Reply:
Done. Do you need me to manually sqash the commits?

Reply:
yes please.

while you are here, could you please also run `gofmt` from 1.17+ on these 2 files? go 1.17 introduced new build constraint format, and `gofmt` 1.17+ would auto generate the new build constraint from the old one. (the new one starts with `//go:build` instead of `// +build`, after running `gofmt` both should be there).

Reply:
I ran gofmt on the affected files. It also kept the original build tags, which I guess is okay to be compatible with go 1.16 (as stated by the [go.mod](https://github.com/apache/thrift/blob/master/go.mod) file)